playground/multibrand-site/__tests__/dev.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { test, expect } from "@playwright/test";
+
+const waitForStyling = async () => {
+  await new Promise((r) => setTimeout(r, 500));
+};
+
+test("index page loads", async ({ page }) => {
+  await page.goto("/");
+  await waitForStyling();
+  await expect(page).toHaveTitle(/Pages Development Page/);
+  await expect(page).toHaveScreenshot();
```

Comment:
I only thought of this know but I think it would be good to check the contents of the index page (at least a little bit). I was thinking we could just add a single check that a link to /sunglasses/227213256(...) exists